tests/bpf2c_tests/expected/bindmonitor_dll.txt

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation
```

Comment:
Why did this line change?

Reply:
No idea. maybe conversion to UTF-8 caused this?

Reply:
Conversion to UTF8 caused extra character to be added at the start. Switching to ASCII fixed it.